### ISSUE TITLE
CIP 36 - Mutual Credit Protocol - Discussions

### ISSUE THREAD

**Issue by YazzyYaz** [celo-org/celo-proposals#168]
Discussions for [CIP 36](https://github.com/celo-org/celo-proposals/pull/151)

**Comment by zviadm** [comment]
hey @dukejones , this is pretty interesting stuff as a concept, but I am a bit confused why this needs to be a CIP? 

Does it need any changes in core Celo or even any type of approval from Celo community?

Is there anything blocking from launching the project and to actually experiment with this in live network?

**Comment by YazzyYaz** [comment]
@zviadm this is a Ring 3 category CIP, so like ERC-20 spec, it doesn't modification to the core, but it's still welcome as a submission since Duke and his team want to look to standardize what a Mutual Credit smart contract would look like for Celo so eveyrone can reference that standard.

**Comment by dukejones** [comment]
Yes, we expect multiple communities and enterprises to launch mutual credit tokens. We want it to be easy to launch a mutual credit token that's broadly compatible, especially so we can design a system for universal multi-currency mutual credit trade networks.
